lib/systems/default.nix

```diff
@@ -50,6 +50,7 @@ rec {
         else if final.isFreeBSD             then "fblibc"
         else if final.isNetBSD              then "nblibc"
         else if final.isAvr                 then "avrlibc"
+        else if final.isGhcjs               then null
```

Comment:
Did you look at it in the context of its commit, 9fa63d3756cc83e14c6ceac9d80e4bfd036a593f?  The commit removes special handling of GHCJS in `libcCrossChooser` so it can be used by other `libc = null` targets as well, and this is part of that refactoring.